src/test/java/com/puppetlabs/jenkins/plugins/puppetgatling/steps/PuppetGatlingArchiverStepTest.java

```diff
@@ -50,6 +50,15 @@ public void archive() throws Exception {
                 1, buildActions.size());
         PuppetGatlingBuildAction pgba = buildActions.get(0);
 
+        String simDir = "simulations/my-sim-1470844250439";
+        File origSimulationLog = new File(b.getRootDir(), simDir + "/simulation.log");
+        File compressedSimulationLog = new File(b.getRootDir(), simDir + "/simulation.log.gz");
+
+        assertFalse("Original simulation log file '" + origSimulationLog + "' should not exist",
+                origSimulationLog.exists());
+        assertTrue("Compressed simulation log file '" + origSimulationLog + "' should exist",
```

Comment:
I think this should be

```
assertTrue("Compressed simulation log file '" + compressedSimulationLog + "' should exist",
```

That is, `compressedSimulatonLog` rather than `origSimulationlog` in the assertion text.